lib/Frontend/ArgsToFrontendOutputsConverter.cpp

```diff
@@ -745,6 +745,16 @@ SupplementaryOutputPathsComputer::readSupplementaryOutputFileMap() const {
           hadError = true;
         }
         outputPaths.push_back(createFromTypeToPathMap(mapForInput));
+
+        // HACK: -emit-fixits-path is frontend-only, so wire it in explicitly
+        // if it was present on the command line.
+        auto &paths = outputPaths.back();
+        if (auto fixitPath = Args.getLastArg(options::OPT_emit_fixits_path)) {
+          if (paths.FixItsOutputPath.empty()) {
```

Comment:
Why this check? The `if (auto ... = Args.getLastArg(options::OPT_emit_fixits_path))` should get just the last entry.

Reply:
Presumably, the supplementary outputs file could have an entry here, so we're saying that would take precedence over the command-line argument.